cypari2/gen.pyx

```diff
@@ -4512,18 +4844,21 @@ cdef Gen list_of_Gens_to_Gen(list s):
 
     This is called from :func:`objtogen` to convert iterables to PARI.
 
-    TESTS::
-
-        sage: from six.moves import range
-        sage: from sage.libs.cypari2.gen import objtogen
-        sage: objtogen(range(10))
-        [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
-        sage: objtogen(i**2 for i in range(5))
-        [0, 1, 4, 9, 16]
-        sage: objtogen([pari("Mod(x, x^2+1)")])
-        [Mod(x, x^2 + 1)]
-        sage: objtogen([])
-        []
+    Tests:
+
+    >>> from six.moves import range
```

Comment:
I just realised this doctests adds a useless dependency on six. Without the import, the doctest works just fine in Py2&3.
However, I'm not sure about the original intent of the test. Maybe replacing `range(10)` with `list(range(10))` is closer to it.
Opinions? I shall start a separate pull request for this.